ext/extreme_timeout/extreme_timeout.c

```diff
@@ -82,10 +82,15 @@ timeout_cb(VALUE block)
     return rb_funcall(block, rb_intern("call"), 0);
 }
 
-VALUE
+/*
+ * XXX: Exposing this function name "timeout" caused segmentation
+ * fault in some environment (See PR #6). Instead of adding "static"
```

Comment:
The sentence "Instead of ..." is not needed. Adding a note about exposure is enough.